### PR TITLE
Now release will first build and test before generating tags

### PR DESCRIPTION
**Pull request by piyush1594**
This will fix the issue of `to run test before generating tags` during release
#1195

### CODE REVIEW

**Comment by piyush1594** [comment]
[test]

**Comment by piyush1594** [comment]
ok to test

**Comment by piyush1594** [comment]
add to whitelist

**Comment by piyush1594** [comment]
[ok to test]

**Comment by rohanKanojia** [comment]
Thanks a lot for your efforts :1st_place_medal: . This looks good to me, just minor nitpick; Could you please use a consistent comment format across this file?

**Comment by piyush1594** [comment]
Thanks a lot @rohanKanojia  Changes are done. :+1:

**Comment by piyush1594** [comment]
Here is the build link https://jenkins.cd.test.fabric8.io/view/all/job/fmp%20Piyush/4/ which got passed.

File: Jenkinsfile [hunk 1]
```diff
@@ -18,58 +18,67 @@
...
+                // Disabled for now as it probably doesn't work because of the different directory structure
+                // with a dedicated doc-module
+                // stage 'Website'
+                // pipeline.website(stagedProject)
+
+                stage('Update downstream dependencies') {
+
+                    // This will update the version of fmp
+                    // in the pom.xml of some downstream repos like
+                    // fabric8-services, quickstart etc.
```
Comment (hrishin) [inline Jenkinsfile]: Does it update the quickstart projects?

Comment (piyush1594) [reply Jenkinsfile]: Here is the list https://github.com/fabric8io/fabric8-maven-plugin/blob/master/release.groovy#L50

Comment (piyush1594) [reply Jenkinsfile]: This test is working right now also, but to run it in Jenkins file this change is required. If we do this change, then will this Jenkins file will work fine.

**Comment by rohanKanojia** [comment]
@hrishin : Shall we merge this and proceed with release?

**Comment by rohanKanojia** [comment]
I'm merging this. Let's test this on main cd pipeline.

**Pull request merged**